plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergPlugin.java

```diff
@@ -196,4 +206,29 @@ private static ConnectorFactory getConnectorFactory()
     {
         return getOnlyElement(new IcebergPlugin().getConnectorFactories());
     }
+
+    @Test
+    public void testNessieMetastore()
+    {
+        ConnectorFactory factory = getConnectorFactory();
+
+        factory.create(
+                        "test",
+                        Map.of(
+                                "iceberg.catalog.type", "nessie",
+                                "iceberg.nessie.warehouse", "/tmp",
+                                "iceberg.nessie.uri", "http://localhost:19120"),
```

Comment:
nit: for consistency sake  to the other tests from this class, we could use `foo` instead of `localhost`